runner/file.go

```diff
@@ -507,6 +508,40 @@ func transformYAMLFile(in string, out *ResultWriter) error {
 	return transformYAML(r, out)
 }
 
+func transformLogFmt(in *bufio.Reader, out *ResultWriter) error {
+	dec := logfmt.NewDecoder(in)
+
+	for dec.ScanRecord() {
+		o := map[string]any{}
+		for dec.ScanKeyval() {
+			if dec.Key() != nil {
+				o[string(dec.Key())] = string(dec.Value())
```

Comment:
Let's keep a map of dec.Key() as a byte to dec.Key() as a string so we don't have to call string(dec.Key()) every time since string() is also an allocation.

Reply:
A slice of bytes cannot be used as key in a map in Go, so probably what you meant was hashing the slice of bytes and mapping it string(dec.Key()).

Reply:
72e445e7651a10fccfe9165413bdc03024e1347d